src/main/java/pl/softwareforge/controller/AppViewController.java

```diff
@@ -0,0 +1,21 @@
+package pl.softwareforge.controller;
+
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class AppViewController
+        implements Initializable {
+
+    @FXML
+    Label labelHello;
+
+    @Override
+    public void initialize(URL url, ResourceBundle resourceBundle) {
```

Comment:
You are not using this function. You broke SOLID rules, please reconsider adding some code, or removing this function and import.